libsplinter/src/network/connection_manager/pacemaker.rs

```diff
@@ -22,6 +22,11 @@ use std::time::Duration;
 
 use crate::network::connection_manager::error::ConnectionManagerError;
 
+/// Pacemaker is responsible for periodically sending a message to
+/// the connection manager that instructs it to send a heartbeat
```

Comment:
This has actually been generalized to send a message to any sender on an interval.  This struct isn't responsible for any of the connection manager's behaviour on each clock tick.

Reply:
Does anything else other than ConnectionManager instantiate and make use of a Pacemaker?

Reply:
Not yet.  But that doesn't mean the doc for this struct should describe connection manager internals.  Connection manager's documentation should describe that it does periodic heartbeats.

Reply:
s/to the connection manager...heartbeat/over a channel.

Reply:
Appended the phrase "over a channel"

Reply:
Appended phrase "over a channel"